hack/cluster-sync.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -ex
+
+${KUBECTL} apply -f deploy/crds/nmstate_v1alpha1_nodenetworkstate_crd.yaml
+${KUBECTL} apply -f deploy/crds/nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml
+${KUBECTL} delete --ignore-not-found -f ${local_handler_manifest}
+${KUBECTL} create -f ${local_handler_manifest}
+
+desiredNumberScheduled="$(${KUBECTL} get daemonset -n nmstate nmstate-handler -o=jsonpath='{.status.desiredNumberScheduled}')"
+for i in {60..0}; do
```

Comment:
let's do 300..0 and set the sleep to 1 second, to save some calculation in code-readers head :)